continuous_integration/recipe/meta.yaml

```diff
@@ -37,6 +39,8 @@ requirements:
     - pygments
     - nest-asyncio
     - tabulate
+    - setuptools-rust>=1.1.2
+    - pyarrow>=4.0.0
```

Comment:
I wouldn't add this to the conda recipe just yet - I'd ideally like to have more discussion around what package we want to consider "core" to the dask-sql conda package first (see https://github.com/dask-contrib/dask-sql/issues/435).

For now, I think just bumping the min version in the:

- docker compose files
- CI environments
- setup.py

Should be sufficient.